PUBLISHING.md

```diff
@@ -0,0 +1,219 @@
+# Publishing to PyPI
+
+This guide covers how to publish the Qiskit MCP servers to PyPI, both manually and via automated workflows.
+
+## Packages
+
+This repository contains two separate PyPI packages:
+
+1. **qiskit-code-assistant-mcp-server** - MCP server for Qiskit Code Assistant
+2. **qiskit-ibm-runtime-mcp-server** - MCP server for IBM Quantum Runtime
+
+## Automated Publishing (Recommended)
+
```

Comment:
should this approach also contain a prerequisites sections? primarily for the GitHub CLI (`gh`).